test/Strings/fromCodePoint.js

```diff
@@ -0,0 +1,22 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+function f()

```

Comment:
Nit: for JS, we should probably stick with not having braces on new lines, because of ASI.